packages/local-storage-discovery/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@waku/local-storage-discovery",
+  "version": "0.0.1",
+  "description": "Local Storage Discovery acts as a way to fetch (healthy) peers that you have previously successfully connected to, and act as a faster way to bootstrap your way into the network.",
```

Comment:
nit: "Local Storage Discovery keeps records of healthy peers in the browser's local storage for quicker bootstrapping."